### ISSUE TITLE
Provide exact match search option at the translate filter that applies to the contains input field

### ISSUE THREAD

**Issue by lourdas** [backdrop-ops/localization#50]
It would really make the translator's job easier if there was a additional filter option (as a checkbox) for exact match of the contains input field, something like this:

![image](https://github.com/backdrop-ops/localize.backdropcms.org/assets/397916/fcd7ace0-b889-44a5-bd0f-92543c919f7a)

Use case: I want to search for the word `field`, but since this appears hundreds of time in the source strings, I cannot find it easily.

Thank you.

**Comment by olafgrabienski** [comment]
Yep, such a filter option sounds useful. Technically, the feature would probably be provided by https://github.com/backdrop-contrib/l10n_server.

**Comment by argiepiano** [comment]
@lourdas can you please post this at https://github.com/backdrop-contrib/l10n_server/issues

**Comment by lourdas** [comment]
> @lourdas can you please post this at https://github.com/backdrop-contrib/l10n_server/issues

I see that you already created an issue there, do you need me to do anything else?

**Comment by argiepiano** [comment]
No more needed. Thanks

**Comment by lourdas** [comment]
I created a PR at https://github.com/backdrop-contrib/l10n_server/pull/34.
